javascript/web/util.js

```diff
@@ -0,0 +1,9 @@
+export const objectToJsonString = obj => {
+    /*
+        Workaround because JS built-in function is not able to convert BigInt
+        https://github.com/GoogleChromeLabs/jsbi/issues/30
+    */
+    BigInt.prototype.toJSON = function() { return this.toString() }
```

Comment:
This can be called only once and don't need to be within `objectToJsonString`. We could refactor this be doing:

1. Rename function to `registerBigIntToJSON()` that now returns nothing and only do `BigInt.prototype.toJSON = function() { return this.toString() }`
1. Import the `registerBigIntToJSON` function in `main.js` and just after all import, add `registerBigIntToJSON()`. 
1. Now `JSON.stringify` will work out of the box.